deploy/obcluster.yaml

```diff
@@ -14,16 +14,10 @@ spec:
   topology:
     - zone: zone1
       replica: 1
-      nodeSelector:
```

Comment:
Don't we need nodeSelector any more?

Reply:
node selector is optional，you can use it to specific certain nodes to run the observer pods, if it's not set, it depends on the K8s scheduler, since this is an example, I'd like to make as simple as possible.